dev/examples/controls/pairedResult.js

```diff
@@ -116,6 +116,40 @@ export const renderPairedResultTimeseriesDateline = (id) => {
     );
     return pairedTime;
 };
+
+export const renderPairedResultTimeseriesEventline = (id) => {
+    const pairedTimeDateline = utils.deepClone(
+        getDemoData(`#${id}`, "PAIRED_TIMESERIES")
+    );
+    pairedTimeDateline.eventline = [
+        {
+            color: "grey",
+            style: {
+                strokeDashArray: "4,4"
+            },
+            value: new Date(2016, 8, 1, 8).toISOString()
+        },
+        {
+            color: "black",
+            style: {
+                strokeDashArray: "2,2"
+            },
+            value: new Date(2017, 5, 1, 12).toISOString()
+        }
+    ];
+    pairedTimeDateline.clickPassThrough = {
+        dateline: false
```

Comment:
Ensure `clickPassThrough` works properly.

Reply:
Since we don't have any onClick property I have removed clickPassThrough property.